src/components/v2/V2Project/V2ProjectReconfigureModal/hooks/reconfigureFundingCycle.ts

```diff
@@ -34,10 +41,17 @@ export const useReconfigureFundingCycle = ({
       throw new Error('Error deploying project.')
     }
 
+    const fundingCycleMetadata = nftRewardsCids?.length
```

Comment:
is there *any* chance that the overrides could kick in without nft rewards?

e.g what if someone has rewards, turns them off. is nftRewardsCids.length 0 in that case?

Reply:
Good point. If it's not then it's a mistake with the contracts. `nftRewardsCids` comes from a `dataSource.allTiers()` call. If the NFT rewards are not active then this surely won't return anything